x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/MultivalueDedupeBoolean.java

```diff
@@ -63,6 +63,32 @@ public BooleanBlock dedupeToBlock(BlockFactory blockFactory) {
         }
     }
 
+    public BooleanBlock sortToBlock(BlockFactory blockFactory, boolean ascending) {
```

Comment:
Could you add the javadoc that you added in https://github.com/elastic/elasticsearch/pull/106095/files#diff-24622e967d2f71d7d290e43f3bc889a1fd65a77c707642916e3912a6773d15a9R186 here as well?